crates/task-impls/src/events.rs

```diff
@@ -94,17 +94,20 @@ pub enum HotShotEvent<TYPES: NodeType, I: NodeImplementation<TYPES>> {
     TransactionsRecv(Vec<TYPES::Transaction>),
     /// Send transactions to the network
     TransactionSend(TYPES::Transaction, TYPES::SignatureKey),
+    /// Transactions have been sequenced
+    TransactionsSequenced(
+        TYPES::BlockPayload,
+        <TYPES::BlockPayload as BlockPayload>::Metadata,
+        Vec<u8>,
```

Comment:
```suggestion
```

I missed this earlier comment https://github.com/EspressoSystems/HotShot/pull/2077#discussion_r1397951288 .

I don't think we should include the encoded txs in this event. If we do this then we're sending 2 copies of the block payload. Anyone receiving this event can get the encoded bytes if they want it simply by calling `encode`.

Reply:
I agree; I think before we change the payload and header items this makes the most sense. @shenkeyao what do you think?

Reply:
Okay, sounds good. I'll put this one on hold until that PR is merged

Reply:
> Oh looks like @ggutoski and I were suggesting different things. Gus was suggesting removing the encoded txns, by I the payload.
> 
> I think the reason we change the `BlockReady` event to `TransactionsSequenced` is that we want to handle the txns rather than the block (payload), so here we should use the encoded txns?

I'm not sure whether to keep the bytes or the payload, but I know we don't want both. In discussion elsewhere it seems like we prefer to keep the bytes everywhere but I'm not yet confident enough to express a strong preference.